examples/docking-protein-homotrimer/docking-protein-homotrimer.cfg

```diff
@@ -0,0 +1,127 @@
+# ====================================================================
+# Rigid-body docking example
+
+# directory in which the scoring will be done
+run_dir = "run1"
+ncores = 20
+# cns_exec = "path/to/bin/cns" # optional
+
+# molecules to be docked
+molecules =  [
+    "data/1qu9_A.pdb",
+    "data/1qu9_B.pdb",
+    "data/1qu9_C.pdb",
+    ]
+
+# ====================================================================
+# Parameters for each stage are defined below, prefer full paths
+#####################################################################
+# WARNING: THE PARAMETERS HERE ARE ILLUSTRATIVE
+#  THE WORKFLOW IS WORK-IN-PROGRESS
+#####################################################################
+[topoaa]
+autohis = true
+
+[rigidbody]
+ambig_fname = 'data/1qu9_whiscy_air.tbl'
+sampling = 20
+noecv = true
+ncs_on = true
+numncs = 2
+ncs_sta1_1="2"
+ncs_end1_1="128"
+ncs_seg1_1="A"
+ncs_sta2_1="2"
+ncs_end2_1="128"
+ncs_seg2_1="B"
+ncs_sta1_2="2"
+ncs_end1_2="128"
+ncs_seg1_2="B"
+ncs_sta2_2="2"
+ncs_end2_2="128"
+ncs_seg2_2="C"
+sym_on = true
+numc3sym = 1
+c3sym_sta1_1="2"
+c3sym_end1_1="128"
+c3sym_seg1_1="A"
+c3sym_sta2_1="2"
+c3sym_end2_1="128"
+c3sym_seg2_1="B"
+c3sym_sta3_1="2"
+c3sym_end3_1="128"
+c3sym_seg3_1="C"
+
+
+[caprieval]
+reference = 'data/1qu9_ABC.pdb'
+
+[seletop]
+select = 5
+
+[flexref]
+ambig_fname = 'data/1qu9_whiscy_air.tbl'
+noecv = true
+ncs_on = true
+numncs = 2
+ncs_sta1_1="2"
+ncs_end1_1="128"
+ncs_seg1_1="A"
+ncs_sta2_1="2"
+ncs_end2_1="128"
+ncs_seg2_1="B"
+ncs_sta1_2="2"
+ncs_end1_2="128"
+ncs_seg1_2="B"
+ncs_sta2_2="2"
+ncs_end2_2="128"
+ncs_seg2_2="C"
+sym_on = true
+numc3sym = 1
+c3sym_sta1_1="2"
+c3sym_end1_1="128"
+c3sym_seg1_1="A"
+c3sym_sta2_1="2"
+c3sym_end2_1="128"
+c3sym_seg2_1="B"
+c3sym_sta3_1="2"
+c3sym_end3_1="128"
+c3sym_seg3_1="C"
+
+[caprieval]
+reference = 'data/1qu9_ABC.pdb'
+
+[emref]
+ambig_fname = 'data/1qu9_whiscy_air.tbl'
+noecv = true
+ncs_on = true
+numncs = 2
+ncs_sta1_1="2"
+ncs_end1_1="128"
+ncs_seg1_1="A"
+ncs_sta2_1="2"
+ncs_end2_1="128"
+ncs_seg2_1="B"
+ncs_sta1_2="2"
+ncs_end1_2="128"
+ncs_seg1_2="B"
+ncs_sta2_2="2"
+ncs_end2_2="128"
+ncs_seg2_2="C"
+sym_on = true
+numc3sym = 1
+c3sym_sta1_1="2"
+c3sym_end1_1="128"
+c3sym_seg1_1="A"
+c3sym_sta2_1="2"
+c3sym_end2_1="128"
+c3sym_seg2_1="B"
+c3sym_sta3_1="2"
+c3sym_end3_1="128"
+c3sym_seg3_1="C"
```

Comment:
I see we have duplicated parameters here among modules. We can add a `take_from_previous = true` kind-of parameter to capture parameters from the previous modules. For example, the hierarchy could be:

`defaults` -> `previous module` -> `current module`.

**Only** if `take_from_previous = true`

Reply:
But you might only want part of the parameters to be taken over - so adds complexity...

Reply:
If that is the case, so better to keep copy+pasting until the need actually arises. I thought the situation was more strict.